ddtrace/bootstrap/sitecustomize.py

```diff
@@ -2,37 +2,32 @@
 Bootstrapping code that is run when using the `ddtrace-run` Python entrypoint
 Add all monkey-patching that needs to run by default here
 """
-import logging
-import os
 import sys
-from typing import Any
-from typing import Dict
 
 
-# Perform gevent patching as early as possible in the application before
-# importing more of the library internals.
-if os.environ.get("DD_GEVENT_PATCH_ALL", "false").lower() in ("true", "1"):
```

Comment:
Have we considered keeping this flag for users who depend on it? It might be worth deprecating it rather than removing it entirely to provide a better upgrade experience. While we are aiming to fix an issue, this flag could be seeing usage for other use-cases that we aren't aware of. 

If we do decide to remove it, we have to be as loud as possible with an `upgrade` note to make it clear what is happening and what action the user has to take.

Reply:
I've updated the diff to leave the flag untouched.